openmp/runtime/src/kmp_barrier.h

```diff
@@ -21,7 +21,10 @@
 #define KMP_ALIGNED_ALLOCATE(size, alignment) _mm_malloc(size, alignment)
 #define KMP_ALIGNED_FREE(ptr) _mm_free(ptr)
 #elif KMP_HAVE_ALIGNED_ALLOC
-#define KMP_ALIGNED_ALLOCATE(size, alignment) aligned_alloc(alignment, size)
+#define KMP_ALGIN_UP(val, alignment)                                           \
+  (((val) + (alignment)-1) / (alignment) * (alignment))
```

Comment:
Is this formatted correctly?

Reply:
I think so. The diff is after `clang-format`,

Reply:
Hmm, that must be a bug in `clang-format`. I did add spaces before and after `-` and `1`.